Maestro/subscriptions.json

```diff
@@ -718,6 +718,8 @@
     // Mirror github changes to devdiv Azure DevOps
     {
       "triggerPaths": [
+	"https://github.com/aspnet/librarymanager/blob/master/**/*",
```

Comment:
I think you want this in the "dnceng" section.

Reply:
FYI if the repo is in DevDiv, the repo name needs to be something like "AspNet-LibraryManager-Trusted" IIRC.

Reply:
I picked DevDiv because this repo belongs to the WebTools team, not the ASP.NET runtime team (though it's under the aspnet GH org), and all of our infrastructure is under DevDiv.  I didn't want this to be the only asset we have under dnceng, as that would generate inconsistency for us.

I've renamed the DevDiv repo to have the -Trusted suffix.

Reply:
aspnet/websdk is unrelated to webtools, right?

Reply:
I think websdk is also owned by our team, but ships with .NET Core.  Library Manager ships only with VS (and a few NuGet packages, but only on the VS ship cycle).